lib/db.js

```diff
@@ -35,7 +35,30 @@ export default (connection_string) => {
               : resolve(result)
           ))
         )
-      );
+      )
+      .catch(err => {
+        const { message, position } = err;
+        if (message && position >= 0) {
+          const endLineWrapIndexOf = string.indexOf('\n', position);
+          const endLineWrapPos = endLineWrapIndexOf >= 0 ? endLineWrapIndexOf : string.length;
+          const stringStart = string.substring(0, endLineWrapPos);
+          const stringEnd = string.substr(endLineWrapPos);
+          const startLineWrapPos = stringStart.lastIndexOf('\n') + 1;
+          const padding = ' '.repeat(position - Math.max(startLineWrapPos, 0) - 1);
```

Comment:
👍 
Position should be >=1 because SQL indexing starts with 1, and Math.max is really redundant.